oak-lucene/pom.xml

```diff
@@ -471,5 +471,11 @@
       <version>1.19.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
It seems we're adding unneeded dependencies here...

Reply:
oak-lucene uses the MongoConnectionFactory in IT(s) [https://github.com/apache/jackrabbit-oak/blob/trunk/oak-lucene/src/test/java/org/apache/jackrabbit/oak/plugins/index/lucene/directory/ActiveDeletedBlobCollectionIT.java] which in turns uses MongoDockerRule, test container dependency has to be added in oak-lucence

Reply:
But isn't that automatically inherited?

Reply:
No, testcontainers dependency is defined under dependencyManagment of oak-parent. This will not be inherited by the child module until it is explicitly defined under dependencies of child module.

Other workaround for this is to define it under the dependencies tag in parent pom, in this way it will be available to all the child modules but this should not be done since this is not needed by all the child modules

Reply:
I was/am confused because I would have assumed that switching from one docker framework to another one will *replace* a dependency (not add one)...